nailgun/entities.py

```diff
@@ -844,9 +844,11 @@ def __init__(self, server_config=None, **kwargs):
             'command': entity_fields.StringField(
                 required=True,
                 str_type='latin1',
+                default='top',
```

Comment:
@elyezer has a good point. In the other cases where we provide default values, those defaults work 100% of the time. How about [this](https://github.com/SatelliteQE/robottelo/pull/2692#issuecomment-132203235) solution?